test/filefetcher_test.py

```diff
@@ -22,11 +22,13 @@
 from hammertime import HammerTime
 from hammertime.http import Entry, StaticResponse
 from hammertime.ruleset import StopRequest, RejectRequest
+from hammertime.kb import KnowledgeBase
 
 from tachyon.core.filefetcher import FileFetcher
 from fixtures import async, create_json_data, FakeHammerTimeEngine, SetResponseCode, fake_future, SetResponseContent,\
-    RaiseForPaths
-from tachyon.core import database
+    RaiseForPaths, SetFlagInResult
+from tachyon.core import database, conf
+from tachyon.__main__ import setup_hammertime_heuristics
```

Comment:
Strange import, setup_hammertime_heuristics probably should not be in there.

Reply:
It there so the same heuristics are used both in the tests and the production. I think it is more maintainable than duplicating the heuristics setup in the test case, as adding a heuristic that affect the behavior would immediately cause the tests to fail.

Reply:
The issue is not about re-using, just about the location of the code being reused.